certbot-apache/tests/testdata/centos7_apache/apache/httpd/conf.d/ssl.conf

```diff
@@ -110,7 +110,7 @@ SSLCipherSuite HIGH:MEDIUM:!aNULL:!MD5:!SEED:!IDEA
 #   certificate chain for the server certificate. Alternatively
 #   the referenced file can be the same as SSLCertificateFile
 #   when the CA certificates are directly appended to the server
-#   certificate for convinience.
+#   certificate for convenience.
```

Comment:
This typo should not be fixed, because this file is copied verbatim from Apache.

Reply:
Done.